File: VB/WindowsFormsApplication662/MySkinViewInfoRegistrator.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraTab.Registrator

Namespace WindowsFormsApplication662
    Public Class MySkinViewInfoRegistrator
        Inherits SkinViewInfoRegistrator

        Public Sub New()
            MyBase.New()

        End Sub
        Public Overrides ReadOnly Property ViewName() As String
            Get
                Return "MyStyle"
            End Get
        End Property
        Public Overrides Function CreatePainter(ByVal tabControl As DevExpress.XtraTab.IXtraTab) As DevExpress.XtraTab.Drawing.BaseTabPainter
            Return New MySkinTapPainter(tabControl)
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication662/MySkinViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraTab.Registrator;

namespace WindowsFormsApplication662 {
    public class MySkinViewInfoRegistrator : SkinViewInfoRegistrator {
        public MySkinViewInfoRegistrator()
            : base() {

        }
        public override string ViewName {
            get {
                return "MyStyle";
            }
        }
        public override DevExpress.XtraTab.Drawing.BaseTabPainter CreatePainter(DevExpress.XtraTab.IXtraTab tabControl) {
            return new MySkinTapPainter(tabControl);
        }
    }
}
